config/config.go

```diff
@@ -62,11 +62,13 @@ type SectionAndroid struct {
 
 // SectionIos is sub section of config.
 type SectionIos struct {
-	Enabled    bool   `yaml:"enabled"`
-	KeyPath    string `yaml:"key_path"`
-	Password   string `yaml:"password"`
-	Production bool   `yaml:"production"`
-	MaxRetry   int    `yaml:"max_retry"`
+	Enabled     bool   `yaml:"enabled"`
+	KeyPath     string `yaml:"key_path"`
+	DevKeyPath  string `yaml:"dev_key_path"`
+	Password    string `yaml:"password"`
+	Production  bool   `yaml:"production"`
+	Development bool   `yaml:"development"`
```

Comment:
If we only plan to support only production and development, then shouldn't an `if`/`else` on `Production` be enough? Otherwise, if we plan to add more environments, should we consider refactoring to a single, string based variable?

Reply:
This PRs purpose is to support attempting both Sandbox and Prod Pushes. https://github.com/axiomzen/gorush/blob/bc91a205a0bff4239757a36d8bd72a07bef41ca5/gorush/notification_apns.go#L19-L58